great-idea-website/index.html

```diff
@@ -16,48 +16,91 @@
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <div class="container">
+        <nav class="nav">
+            <ul style="display: flex; margin-bottom: 50px;">
+              <li>Services</li>
+              <li>Product</li>
+              <li>Vision</li>
+              <li>Features</li>
+              <li>About</li>
+              <li>Contact</li>
+              <li>
+                  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+              </li>
+            </ul>
+          </nav>
+          
+        <div class="header-text-container">
+            <h1 class="header-text"> Innovation <span class="new-line">On </span> Demand</h1>
+            <div class="text-center">
+                <button class="button">Get Started</button>
+            </div>
+        </div>
+      
+
+        <div class="header-image-container">
+            <img src="img/header-img.png" alt="Image of a code snippet.">
+        </div>       
+        
+      <hr>
+
+
+  <div class="section-2">
+    <h1 class="h1-text">Features</h1>
```

Comment:
Normally you want just a single `<h1>` on your page. That should be your main heading (you can think at it as the "title of the page"). You could substitute all the `<h1>` after the `Innovation On Demand` one with `<h2>` or `<h3>`